github/teams_members.go

```diff
@@ -96,26 +124,32 @@ type TeamAddTeamMembershipOptions struct {
 	Role string `json:"role,omitempty"`
 }
 
-// AddTeamMembership adds or invites a user to a team.
-//
-// In order to add a membership between a user and a team, the authenticated
```

Comment:
Aren't all these comments (from old lines 101 through 114) still valid?
If so, can we please keep them?

Reply:
On the github website it was a bit different. Instead of updating it, I removed it because, for other endpoints, the documentation is not imported as we have the link to the github documentation. 
I can put it back with the updated description if you want.

Reply:
Oh, OK... gotcha.  Thanks, @adrienzieba... I'm fine either way, actually.
You are right that the other endpoints don't have it and we do link to the main docs, so that is probably sufficient.